### PR TITLE
Use `display-color-cells` to enable 24-bit color terminal support

### PR DESCRIPTION
**Pull request by visigoth**
This change enables the use of nord-theme with emacs' new 24-bit color terminal support.

### CODE REVIEW

**Comment by christopherraa** [comment]
I see this PR has been here for a while. Are there someone with write access here that could do a review? It would be nice to see if this PR solves #64 as well.

**Comment by rien333** [comment]
@arcticicestudio

**Comment by arcticicestudio** [comment]
Sorry for the enormous delays, but "real life" keeps me busy almost ~18h a day 😟
Like described in https://github.com/arcticicestudio/nord-emacs/issues/59#issuecomment-447625096, I heavily rely on the community for this port project since I'm not a Emacs user and I resist to learn the language of syntactically arranged brace characters (Lisp) 😝

Anyway, I promise to review, reproduce and test this PR no later than Saturday afternoon. I've seen a lot of forks of this repository only for this fix and many users approved the problems was resolved using the changes from this PR so I guess it'll work we way it should without breaking any other users setup.

File: nord-theme.el [hunk 1]
```diff
@@ -450,6 +454,11 @@
     `(web-mode-variable-name-face ((,class (:foreground ,nord-variable))))
 
     ;; +--- UI ---+
+
+    ;; > ace-jump-mode
```
Comment (arcticicestudio) [inline nord-theme.el]: Please remove this additional style block and maybe submit a new PR if you'd like to add support for the package to Nord.

File: nord-theme.el [hunk 2]
```diff
@@ -480,6 +489,19 @@
    `(diff-hl-change ((,class (:background ,nord13))))
    `(diff-hl-insert ((,class (:background ,nord14))))
    `(diff-hl-delete ((,class (:background ,nord11))))
+
+    ;; > diredp
```
Comment (arcticicestudio) [inline nord-theme.el]: Same like above: Please also remove this additional style block.

**Comment by arcticicestudio** [comment]
@visigoth I can also go ahead and rebase your PR on `develop` and remove both unrelated code blocks if you like, just let me know so we can get this finally merged :smile:

**Comment by arcticicestudio** [comment]
Since the source branch of this PR is not a _feature_ branch but the default branch [visigoth/nord-emacs@develop][r] there's no way for me to update the required PR changes. I'll create a new branch based on this PR and open a new PR to superseed this one. 

[r]: https://github.com/visigoth/nord-emacs

**Comment by arcticicestudio** [comment]
Superseded by #88.

**Pull request closed without merging**